src/api/Annotations.js

```diff
@@ -164,16 +192,84 @@ export default class Annotations extends MarkerBasedApi {
             return;
         }
 
+        const requestData = {
+            file_id: fileId,
+            file_version_id: fileVersionId,
+            ...(shouldFetchReplies ? { fields: 'replies' } : null),
+        };
+
         this.markerGet({
             id: fileId,
             errorCallback,
             limit,
-            requestData: {
-                file_id: fileId,
-                file_version_id: fileVersionId,
-            },
+            requestData,
             shouldFetchAll,
             successCallback,
         });
     }
+
+    getAnnotationReplies({
```

Comment:
Can we use parameters rather than an object to stay consistent with the other methods in this class?